src/utils/config_parser/default.test.ts

```diff
@@ -0,0 +1,7 @@
+import { getDefaultOwnershipRules } from "./default";
+
+describe("Rule matcher tests", () => {
+  test("Default rules have correct content", () => {
+    expect(getDefaultOwnershipRules().directoryMatchingRules[0].path).toMatch("playground/{{username}}/**/*");
```

Comment:
Codacy found an issue: ['magic numbers' are not allowed: 0](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6092258)